js/react_native/android/build.gradle

```diff
@@ -20,6 +20,25 @@ def getExtOrIntegerDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : (project.properties['OnnxruntimeModule_' + name]).toInteger()
 }
 
+def getOnnxruntimePackageName() {
+  // locate user's project dir
+  def reactnativeRootDir = project.rootDir.parentFile
+  // get package.json file in root directory
+  def packageJsonFile = new File(reactnativeRootDir, 'package.json')
+  // read field 'ortPackageName'
+  if (packageJsonFile.exists()) {
+    def packageJsonContents = packageJsonFile.getText()
+    def packageJson = new groovy.json.JsonSlurper().parseText(packageJsonContents)
+    def packageName = packageJson.ortPackageName
```

Comment:
what happens if the field is not there?

Reply:
I tested the case when user specifies nothing. so it won't fail and will just return a "". and the ort-ext aar won't be included in that case. 

maybe there's a better check for this.